packages/@aws-cdk/aws-codepipeline-aws-actions/package.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
Sadly, there is no support for “re-exporting” symbols in jsii, which means we can’t vend aws-actions to all languages. What I suggest is providing pointers in the codepipeline README to all action types. This will be where people will naturally look for them.

Reply:
What if we kept the `aws-codepipeline-aws-actions` module in other languages, but remove the re-exporting part? This way, it would leave it as only depending on the other Action modules, and in this way make it a little easier for customers to just import all of them in one go. Does that have enough value?

Also, any comments on the changed names of the Actions?

Reply:
aws-actions feels convoluted and confusing... how about we start without, and see if it's needed?
No comments on the the action names - 👍